tests/e2e/general_day_two_test.go

```diff
@@ -68,4 +69,74 @@ var _ = Describe("TF day2 scenrios", func() {
 				Expect(resourceStateMap["organization_name"]).To(Equal(currentAccountInfo.Body().Organization().Name()))
 			})
 	})
+	Context("Author: smiron-Medium-OCP-64907 @OCP-64907 @smiron", func() {
+		var (
+			clusterService           *exe.ClusterService
+			err                      error
+			profile                  *ci.Profile
+			originalCustomProperties map[string]string
+		)
+
+		BeforeEach(func() {
+			// Load profile from YAML file based on environment
+			profile = ci.LoadProfileYamlFileByENV()
+
+			// Initialize the cluster service
+			clusterService, err = exe.NewClusterService(profile.ManifestsDIR)
+			Expect(err).ShouldNot(HaveOccurred())
+
+			// Capture the original custom properties
+			originalCustomProperties = map[string]string{
```

Comment:
We should record the original properties by getting cluster detail rather than read from constants. In future we will have other properties added.

Reply:
ack, to fix it, I created a new function that called `ReadTerraformTFVars`,
which will read `terraform.tfvars` file recorded when the test began, and return its content as a map in the BeforeEach. late on, in AfterEach we will restore to the original property by applying the var.